blazor/predefined-dialogs/getting-started.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: post
+title: Getting Started with Blazor Predefined Dialogs Component | Syncfusion
+description: Checkout and learn about getting started with Blazor Predefined Dialogs component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Predefined Dialogs
```

Comment:
Control: Predefined Dialogs

Reply:
In `Breadcrumb` component they have given lower case only. So not corrected this changes.

<img width="662" alt="image" src="https://user-images.githubusercontent.com/93525217/182564491-6280ce19-93a0-4997-94d2-e4708dae4964.png">